### PR TITLE
refactor(swap): modals

### PR DESCRIPTION
**Pull request by banklesss**
Run `./gradlew clean` in the `android` folder

- Currently this solution its only applied to the confirm tx screen, the one with the problematic input.
- I needed to change `windowSoftInputMode` from `adjustPan` to `adjustResize` because it was impossible to fix the modals before this. This config was applied time ago to fix [this](https://emurgo.atlassian.net/browse/YOMO-549). A alternate solution has been applied.
- Tested on real and virtual devices on different screen sizes.

### CODE REVIEW

File: apps/wallet-mobile/src/WalletNavigator.tsx [hunk 1]
```diff
@@ -35,6 +52,9 @@ const WalletTabNavigator = () => {
           tabBarLabelStyle: {fontSize: 11},
           tabBarActiveTintColor: theme.COLORS.NAVIGATION_ACTIVE,
           tabBarInactiveTintColor: theme.COLORS.NAVIGATION_INACTIVE,
+          tabBarStyle: {
+            display: keyboardStatus ? 'none' : undefined,
```
Comment (banklesss) [inline apps/wallet-mobile/src/WalletNavigator.tsx]: Alternate fix for https://emurgo.atlassian.net/browse/YOMO-549

**Review by jorbuedo — APPROVED** [review]

File: apps/wallet-mobile/src/features/Swap/useCases/ConfirmTxScreen/ConfirmTxScreen.tsx [hunk 2]
```diff
@@ -94,32 +87,27 @@ export const ConfirmTxScreen = () => {
               authWithOs()
               return
             }
-            openBottomSheet()
+            openModal(
```
Comment (michalszorad) [inline apps/wallet-mobile/src/features/Swap/useCases/ConfirmTxScreen/ConfirmTxScreen.tsx:90]: Do we need something similar in tx cancellation?

Comment (banklesss) [reply apps/wallet-mobile/src/features/Swap/useCases/ConfirmTxScreen/ConfirmTxScreen.tsx:90]: Its going to be implemented for all swap modals after this PR is merged

**Review by michalszorad — APPROVED** [review]

**Comment by banklesss** [comment]
Working to solve an issue

**Comment by banklesss** [comment]
> Working to solve an issue

issue fixed

**Review by michalszorad — APPROVED** [review]

**Review by SorinC6 — APPROVED** [review]

**Pull request merged**